### PR TITLE
Updated compatibility table and unallowed browsers

### PR DESCRIPTION
**Pull request by edwinchan-microsoft**
@chrfox 
Based on feedback we split the table into 2 parts.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/endpoint-dlp-using.md [hunk 1]
```diff
@@ -46,8 +46,23 @@ Or
 |Unallowed Apps     |Supported         |Supported         |         |
 |Unallowed Bluetooth apps    |Supported         |Not Supported         |         |
 |Browser and domain restrictions to sensitive items      |Supported         |Supported         |         |
-|Additional settings for Endpoint DLP     |Supported         |Supported         |Only the default business justifications are supported for macOS devices         |
 |Always audit file activity for devices     |Supported         |Supported         |         |
+|Auto-quarantine files from unallowed apps     |Supported         |Not supported         |        |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-using.md:50]:
```suggestion
|Auto-quarantine files from unallowed apps     |Supported         |Not Supported         |        |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @edwinchan-microsoft, for consistency, kindly commit the minor suggested change. Many thanks.

**Comment by chrfox** [comment]
the changed content still says "Edge VERSION NUMBER NEEDED"

**Comment by yogkumgit** [comment]
@edwinchan-microsoft Please implement the suggestions. Thanks

**Pull request closed without merging**